src/metabase/driver/presto.clj

```diff
@@ -40,25 +42,34 @@
 
 (defn- parse-time-with-tz [s]
   ;; Try parsing with offset first then with full ZoneId
+  (println "getting ready to parse " s)
```

Comment:
assuming this is left over from debugging? You can leave it there just make it `log/debug` or something